lib/recurly/pager.php

```diff
@@ -45,15 +44,14 @@ public function rewind() {
    */
   public function current()
   {
-    if (empty($this->_count)) {
+    if ($this->count() == 0) {
       return null;
     }
 
     while ($this->_position >= sizeof($this->_objects)) {
       if (isset($this->_links['next'])) {
-        $num_objects = sizeof($this->_objects);
         $this->_loadFrom($this->_links['next']);
-        $this->_position -= $num_objects;
```

Comment:
To me it seemed like an overly complicated way of doing it... I don't see a scenario where we modify the array, we just always replace the records. So I think we can simplify it.